components/ToDoContainer.vue

```diff
@@ -24,8 +81,22 @@ const handleComplete = (id: string) => {
 
     <Search @search="handleSearch" />
 
-    <!-- TaskList :tasks="tasks" @onDeleteTask={handleDelete} @onCompleteTask={handleComplete} -->
-    <TaskList :searchBy="searchBy" />
+    <div class="tasks-header">
+      <div class="tasks-header__total">
+        <NumberBlock title="Total tasks:" text-class="number-text" :number="tasks.length" />
+      </div>
+      <div class="tasks-header__complied">
+        <NumberBlock title="Completed" :number="completedTasks" />
+      </div>
+    </div>
+
+    <TaskList :searchBy="searchBy" :tasks="filteredTasks" @remove-task="handleDeleteTask"/>
+
+    <ModalsRemoveTask v-if="removeTaskModal" @reject-deleting="rejectDeleting" @accept-deleting="acceptDeleting"/>
```

Comment:
I'd prefer to name functions that are passed as callbacks such as **handle*functionName***, in this case **handleRejectDeleting**